package.json

```diff
@@ -16,7 +16,7 @@
     "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
     "start:dev": "export $(cat .env) && npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
     "start-feature": "export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js",
-    "start:e2e": "export $(cat e2e.env) && npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
+    "start:e2e": "export $(cat e2e.env) && export NOTIFICATION_BANNER_START_DATE='2023-09-13' && export NOTIFICATION_BANNER_END_DATE='2050-09-13' && export NOTIFICATION_HEADER='Scheduled Maintenance' && export NOTIFICATION_BODY='This application will be unavailable from 15 September 2023 to 24 October 2023, save your work ahead of time' && npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
```

Comment:
I don't think we need this explicit setting of env vars because they will be taken from the e2e.env file.